src/Rap2hpoutre/LaravelLogViewer/LaravelLogViewer.php

```diff
@@ -18,6 +18,13 @@ class LaravelLogViewer
      */
     private $folder;
 
+    private static $patterns = [
```

Comment:
Sorry about that: the class is not static anymore (just a recent change). Please use `private $patterns` instead of `private static $patterns` and `$this->patterns` vs `self::patterns`